core/modules/comment/templates/comment.tpl.php

```diff
@@ -60,12 +63,14 @@
 ?>
 <article class="<?php print implode(' ', $classes); ?> clearfix"<?php print backdrop_attributes($attributes); ?>>
 
-  <?php print render($title_prefix); ?>
   <?php if ($new): ?>
     <mark class="new"><?php print $new; ?></mark>
   <?php endif; ?>
-  <h3><?php print $title; ?></h3>
-  <?php print render($title_suffix); ?>
+  <?php if ($title_display == TRUE): ?>
```

Comment:
This could be shortened (doesnt have to be so, just saying) to 
`<?php if ($title_display): ?>`

Reply:
Agreed:

```suggestion
  <?php if ($title_display): ?>
```